Objects/complexobject.c

```diff
@@ -34,6 +35,14 @@ _Py_c_sum(Py_complex a, Py_complex b)
     return r;
 }
 
+Py_complex
+_Py_cd_sum(Py_complex a, double b)
```

Comment:
Maybe naming like ``_Py_cr_sum()`` (``r`` - for "real") is better?  As similar helper might be introduced for complex and imaginary arguments (see e.g. [GSL](https://www.gnu.org/software/gsl/doc/html/complex.html#c.gsl_complex_add_imag)).